### PR TITLE
Add command aliases to user guide

### LINKED ISSUE #183: Add short form commands to user guide
Follow up from issue #181

### PR DESCRIPTION
**Pull request by awdse22**
Close #183

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ Sellsavvy is a **desktop app for managing contacts, optimized for use via a Comm
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
+
+* Command aliases are alternative words you can use to execute the same command for convenience.<br>
+  e.g. `listcustomer` has the same function as `listc`
```
Comment (kaixuan477) [inline docs/UserGuide.md:69]: Coudn't comment on line 63 cause its not edited, but `list` is still unchanged.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -304,19 +319,19 @@ _Details coming soon ..._
...
+| **Delete a person**                   | `deletecustomer`, `deletec` | `deletecustomer INDEX`<br> e.g., `deletecustomer 3`                                                                                                                                 |
+| **Edit a person**                     | `editcustomer`, `editc`     | `editcustomer INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g. `editcustomer 2 n/James Lee e/jameslee@example.com`                                         |
+| **Find person(s)**                    | `findcustomer`, `findc`     | `findcustomer KEYWORD [MORE_KEYWORDS]`<br> e.g. `findcustomer James Jake`                                                                                                           |
+| **List all persons**                  | `listcustomer`              |                                                                                                                                                                                     |
```
Comment (kaixuan477) [inline docs/UserGuide.md:329]: Missing `listc`.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -304,19 +319,19 @@ _Details coming soon ..._
...
+| **Revert an order to pending status** | `unmarkorder`, `unmarko`    | `unmarkorder ORDER_INDEX`<br> e.g., `unmarkorder 2`                                                                                                                                 |
+| **Filter orders by status**           | `filterorder`, `filtero`     | `filterorder ORDER_STATUS`<br> e.g. `filterorder Completed`                                                                                                                         |
+| **Help**                              | `help`                      |                                                                                                                                                                                     |
```
Comment (kaixuan477) [inline docs/UserGuide.md:337]: Not sure if necessary, but can include `exit` as well?

**Review by kaixuan477 — COMMENTED** [review]
Minor chnages, looks good.

File: docs/UserGuide.md (hunk 1 above)
Comment (awdse22) [reply docs/UserGuide.md:69]: changed

File: docs/UserGuide.md (hunk 2 above)
Comment (awdse22) [reply docs/UserGuide.md:329]: added

File: docs/UserGuide.md (hunk 3 above)
Comment (awdse22) [reply docs/UserGuide.md:337]: added

**Review by kaixuan477 — APPROVED** [review]
Looks good.

**Pull request merged**